### ISSUE TITLE
Create Getting Started documentation

### ISSUE THREAD

**Issue by syvlabs** [gojek/feast#98]
**Is your feature request related to a problem? Please describe.**
The Getting Started doc will make it easier for new developers to come in, test out, and contribute to Feast.

This doc might include:
- Installation
- Usage
- Dev Environment Setup

**Comment by woop** [comment]
Will pick this up as soon as possible.

**Comment by woop** [comment]
I've started with this. There are a few things that need to be resolved first though.

- We need to add Redis to the Feast helm chart so that users don't need to set it up
- The Python SDK needs to be made available on Pypi (#121)

**Comment by woop** [comment]
I've started this Google Doc for us to contribute to a getting started guide. Would be great to get feedback or contributions from users. After we have settled the guide, we can convert it to markdown

[Feast Getting Started Guide](http://bit.ly/feast-getting-started)

**Comment by thirteen37** [comment]
I'm continuing work on this. Was previously blocked by a lack of a tagged release and corresponding artifacts to reference in the docs.

**Comment by woop** [comment]
Need to pick this up separately for the 0.3.0. Will create a new issue.
